integration_tests/pages/scan/scanBarcodeResult.ts

```diff
@@ -17,6 +17,13 @@ export default class ScanBarcodeResultPage extends Page {
     return Page.verifyOnPage(ScanAnotherBarcodePage)
   }
 
+  isSuccessOrRandomCheck = (): ScanBarcodeResultPage => {
+    cy.get('h1')
+      .invoke('text')
+      .should('match', /(Ready|random)/i)
+    return Page.verifyOnPage(ScanBarcodeResultPage)
+  }
```

Comment:
Needed to cater for random checks being turned on in real envs (though not normally when running integration tests)

Reply:
Yes, makes sense 👍